package.json

```diff
@@ -10,19 +10,20 @@
     "moment": "^2.18.1",
     "node-fetch": "^1.7.0",
     "prop-types": "^15.5.10",
-    "react": "^15.5.4",
-    "react-bootstrap": "^0.31.0",
-    "react-dom": "^15.5.4",
+    "react": "^15.6.1",
+    "react-bootstrap": "^0.31.1",
+    "react-dom": "^15.6.1",
     "react-router-bootstrap": "^0.24.2",
-    "react-router-dom": "^4.1.1"
+    "react-router-dom": "^4.1.2",
+    "whatwg-fetch": "^2.0.3"
```

Comment:
1. Możesz mi odpisać dokładnie jaki miałeś problem z `fetch`-em że musiałeś to dodać.
2. To jest tabelka ze wsparciem dla fetch API:
http://caniuse.com/#search=fetch
Na nasze cele "naukowe" możemy sobie uznać że ignorujemy wsparcie IE. W związku z tym nie powinno się dodawać polyfilli, czy też "przeciążać" ficzerów i api które mamy w przeglądarce. A ten już możemy uznać że mamy (poza IE ;) ) w związku z tym nie wpinaj żadnego fetcha i używaj tego co jest